packages/gexiv2.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gexiv2 < Package
+class Gexiv2 < Meson
   description 'gexiv2 is a GObject wrapper around the Exiv2 photo metadata library.'
   homepage 'https://wiki.gnome.org/Projects/gexiv2/'
   version '0.14.0'
```

Comment:
Update to [0.14.2](https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz)